lib/index.js

```diff
@@ -316,6 +316,53 @@ exports.parseSchemaOrgMicrodata = BBPromise.method(function(chtml){
 });
 
 /**
+ * Scrapes Highwire Press metadata given html object
+ * @param  {Object}   chtml html Cheerio object
+ * @return {Object}   promise of highwire press metadata object
+ */
+exports.parseHighwirePress = BBPromise.method(function(chtml){
+
+	var meta = {};
+	var metaTags = chtml('meta');
+
+	if (!metaTags || metaTags.length === 0) {
+		return BBPromise.reject('No Highwire Press metadata found in page');
+	}
+
+	metaTags.each(function() {
+		var element = chtml(this);
+		var nameAttr = element.attr('name');
+
+		// If the element isn't a Highwire Press property, skip it
+		if (!nameAttr || (nameAttr.substring(0, 9).toLowerCase() !== 'citation_')) {
+			return;
+		}
+
+		var property = nameAttr.substring(nameAttr.indexOf('_') + 1);
```

Comment:
the lower case comment I made before actually referred to this line, so we're ensuring that the result is that the parameters are lower case. for line 337 it doesn't actually matter since you're just determining if the prefix is correct :)